CONTRIBUTING.MD

```diff
@@ -0,0 +1,81 @@
+# Contributing guide
+
+Thanks for your contribution 🥳 to the Fastily website redesign project!
+
+[Code of Conduct](https://github.com/fastify/.github/blob/main/CODE_OF_CONDUCT.md)
+
+* [Vocabulary](#vocabulary)
+* [📝 Commit message guidelines](#-commit-message-guidelines)
+* [📜 Pull Request Policy](#-pull-request-policy)
+* [Before merging](#before-merging)
+* [When merging](#when-merging)
+* [Becoming a collaborator](#becoming-a-collaborator)
+* [🐛 Debugging failing checks](#-debugging-failing-checks)
+* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+
+## 📝 Commit message guidelines
+
+This project follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+Basic rules:
+
+Supported subsystems:
+> build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+
+* Commit messages **must** start with a capital letter
+
+* Commit messages **must not** end with a period `.`
+
+## 📜 Pull Request Policy
+
+### Before merging
```

Comment:
At this stage, this is an overhead for me - the only one who is doing it 🥲

Reply:
I can help you and you are collaborator so you can do fast track

Reply:
I think this is over-engineering translated to documentation
 
I would prefer not to commit to this and keep the fastify org rules at https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md#rules